data/covid19za_provincial_cumulative_timeline_recoveries.csv

```diff
@@ -24,3 +24,4 @@ date,YYYYMMDD,EC,FS,GP,KZN,LP,MP,NC,NW,WC,UNKNOWN,total,source
 15-05-2020,20200515,643,108,1552,795,36,44,20,28,2857,0,6083,https://twitter.com/nicd_sa/status/1261360621804806145/photo/1
 16-05-2020,20200516,746,108,1583,806,37,49,24,28,3097,0,6478,https://twitter.com/nicd_sa/status/1261738963716489219/photo/1
 17-05-2020,20200517,839,108,1583,817,37,49,24,28,3521,0,7006,https://twitter.com/nicd_sa/status/1262110389111791617/photo/1
+18-05-2020,20200518,839,108,1657,818,37,53,27,28,3731,0,7298,https://twitter.com/nicd_sa/status/1262476267405946883
```

Comment:
Could add the `/photo/1` to keep consistent :)

Reply:
Done.